pkg/compute/manifest/manifest.go

```diff
@@ -207,7 +207,7 @@ func (f *File) Read(fpath string) error {
 	f.exists = true
 
 	if f.ManifestVersion.int == 0 {
-		return errors.ErrMissingManifestVersion
+		f.ManifestVersion.int = 1
```

Comment:
This should be
```suggestion
		f.ManifestVersion.int = ManifestLatestVersion
```

Reply:
Ooh good shout!

Reply:
I would disagree actually, if we were to release a v2 of the manifest then it should definitely have `manifest_version` in all new files. So this should always default to 1 if not defined.